### ISSUE TITLE
@vx/bounds: Shouldn't react-dom be a peer dependency?

### ISSUE THREAD

**Issue by denkristoffer** [hshoff/vx#130]
**tl;dr:** Shouldn't `react-dom` be a peer dependency instead of a dependency in `@vx/bounds`? 😄 

I'm building a histogram on a site deployed with [Netlify](https://netlify.com) but their build log is complaining about the following:

`Module not found: Error: Can't resolve 'react/lib/React' in '/opt/build/repo/packages/website/node_modules/@vx/bounds/node_modules/react-dom/lib'`

I've been trying to find the cause of the error with no great success so far since it builds fine on my own machine. I was thinking that it could be due to me using the React 16 beta in this project. Could specifying the `react-dom` package as a peer dependency possibly fix this?

**Comment by hshoff** [comment]
Sorry about that @denkristoffer! will fix with v0.0.137
